src/metabase/config.clj

```diff
@@ -56,8 +56,7 @@
 
 ;; separate map for EE stuff so merge conflicts aren't annoying.
 (def ^:private ee-app-defaults
-  {:embed-max-session-age      "1440"   ; how long a FULL APP EMBED session is valid for. One day, by default
-   :mb-session-cookie-samesite "lax"})
+  {:embed-max-session-age      "1440"}) ; how long a FULL APP EMBED session is valid for. One day, by default
```

Comment:
nit - don't need the extra whitespace after removing the other map value

Reply:
🤦 thank you!